woocommerce/woocommerce-android
# Product creation using package photo: Loading & error states

This PR adds the loading and error UI states. It just shows the loading screen for demonstration purposes to keep the PR smaller. The product title & description generation will be implemented in the next PR. To see the error and no keywords message, please see the previews of `PackagePhotoBottomSheet` in Android Studio.

<img src="https://github.com/woocommerce/woocommerce-android/assets/1522856/9592b147-601e-46f7-aa7e-cd7fab00cb20" width="350" />


**To test:**
1. Go to Products
2. Tap on + FAB
3. Select the AI option
4. Tap on Use package photo button
5. Use the media library to select a picture of a product with some text
6. Notice the image is shown and keywords are extracted
7. Notice the keywords and Regenerate button above are disabled

**Note: ** Please merge #10004 first.
